app/uk/gov/hmrc/thirdpartyapplication/services/ClientSecretService.scala

```diff
@@ -39,7 +38,7 @@ class ClientSecretService @Inject()(config: ClientSecretServiceConfig) {
     val secretValue = clientSecretValueGenerator()
 
     ClientSecret(
-      name = maskSecret(secretValue),
```

Comment:
Can we remove this method as well (it's down below)?

Reply:
Yeah, fair one, it probably can go. It's still used when populating the name field in the response that goes to TPDFE, but I can re-gig that too

Reply:
Ah I thought we didn't have to do this anymore as we had changed the frontend already

Reply:
Done